lib/logging/CBasicLogConfigurator.cpp

```diff
@@ -140,3 +140,10 @@ EConsoleTextColor::EConsoleTextColor CBasicLogConfigurator::getConsoleColor(cons
 	else
 		throw std::runtime_error("Color " + colorName + " unknown.");
 }
+
+void CBasicLogConfigurator::deconfigure() {
```

Comment:
Put `{` on new line.

Reply:
I fill fix these but this is ridiculous. In this time and age it'd be better if you provided proper .clang-format instead of wasting peoples' time on putting brackets manually.

Reply:
I do not think somebody will argue against adding .clang-format The only question who will spend a few hours and add it.

Reply:
> I do not think somebody will argue against adding .clang-format The only question who will spend a few hours and add it.

Somebody who's definitely knowledgeable enough about the style convention here and is willing to save his time in long run by not subjecting himself and other people to code formatting reviews.

Reply:
We tried uncrustify https://github.com/vcmi/vcmi-uncrustify, it appears to be not that easy to setup.

Reply:
Clang-format would be a better option because it's natively supported by Visual Studio and other IDEs too
You just type your stuff and don't worry about code formatting ever again

Reply:
OK, I`ve tried clang-format, it can handle simple cases. Config pushed to vcmi-uncrustify. Use carefully.